static/files/hoprd.cfg.yaml

```
@@ -27,14 +23,9 @@ hopr:
 api:
   enable: true
   host:
     address: !IPv4 0.0.0.0
-    port: 3001
-inbox:
-  capacity: 512
-  max_age: 900
-  excluded_tags:
-    - 0
\ No newline at end of file
+    port: 3001
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)